packages/expo-updates/src/Updates.ts

```diff
@@ -250,6 +250,8 @@ export async function clearLogEntriesAsync(): Promise<void> {
  * storage. This method cannot be used in development mode, and the returned promise will be
  * rejected if you try to do so.
  *
+ * > **Note:** Invoking `fetchUpdateAsync()` to download an update without [`reloadAsync()`](#updatesreloadasync) installs the update on the next app launch.
```

Comment:
```suggestion
 * > **Note:** If a new update is available, invoking this method without calling then [`reloadAsync()`](#updatesreloadasync) upon completion installs the update on the next app cold start.
```

Reply:
Thank you for the better explanation. Will include these new changes.

Reply:
Updated ✅